cmd/portable-package.rb

```diff
@@ -36,6 +36,11 @@ def portable_package
       begin
         deps = Utils.safe_popen_read("brew", "deps", "-n", "--include-build", name).split("\n")
 
+        if deps.include?("glibc@2.13")
+          safe_system "brew", "install", "--ignore-dependencies", *verbose, "glibc@2.13"
+          deps -= ["glibc@2.13"]
+        end
```

Comment:
`linux-headers@4.4` will still be there.

Maybe try something like (variable names could maybe be better):

```suggestion
        deps_to_install_from_bottles = ["glibc@2.13"]
        deps = Dependency.expand(Formula[name], cache_key: "portable-package-#{name}") do |dependent, dep|
          Dependency.prune if dep.test? || dep.optional?

          next if deps_to_install_from_bottles.include?(dep.name)

          Dependency.keep_but_prune_recursive_deps
        end.map(&:name)
        
        bottled_deps, deps = deps.partition { |dep| deps_to_install_from_bottles.include?(dep) }
        
        safe_system "brew", "install", *verbose, *bottled_deps
```